### PR TITLE
Hide end dates and duration for milestones; fixes #1450

### LINKED ISSUE #1450: Can't change planned end date
I can't change planned end date auf this sub task.
No error
No error in Debug mode 
No message

The new changed date is not saved (28-12-2016 / 31-12-2016).
img_036

### PR DESCRIPTION
**Pull request by trasher**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #1450

### CODE REVIEW

File: inc/projecttask.class.php [hunk 1]
```diff
@@ -455,7 +460,7 @@ function showForm($ID, $options=array()) {
                               array('value' => $this->fields['real_start_date']));
       echo "</td></tr>\n";
 
-      echo "<tr class='tab_bg_1'>";
+      echo "<tr class='tab_bg_1 milestone" . ($this->fields['is_milestone'] ? ' starthidden' : '')  . "'>";
```
Comment (orthagh) [inline inc/projecttask.class.php]: milestone class is always defined ?

Comment (trasher) [inline inc/projecttask.class.php]: Yep, this is used in the JS to know which elements to show/hide

Comment (orthagh) [inline inc/projecttask.class.php]: ok, so maybe rename the class to is_milestone or toggle_milestone

Comment (trasher) [inline inc/projecttask.class.php]: ok, done :)

**Pull request merged**